src/main/resources/templates/error/keycloakClientError.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:ng="http://angularjs.org" xmlns:th="http://www.thymeleaf.org" ng-app="pdaApp">
+<head>
+    <meta charset="UTF-8">
+    <title translate="PROJECT_NAME"></title>
+
+    <link rel="icon" href="/common/images/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" type="text/css" href="/common/css/common.css">
+    <link rel="stylesheet" type="text/css" href="/common/css/defaultError.css">
+    <link rel="stylesheet" type="text/css" ng-href="/{{industry}}/css/common.css">
+    <link rel="stylesheet" type="text/css" ng-href="/{{industry}}/css/error.css">
+</head>
+<body>
+    <input type="hidden" id="current_web_service_mode" th:value="${currentWebServiceMode}">
+    <div class="message warning industry_font" ng-controller="error_controller as error">
+        <div class="error_head">
+            <h1>
+                <span class="keycloak_error_title" translate="KEYCLOAK_UNAUTHORIZED_CLIENT_TITLE"></span>
```

Comment:
I think we should take use of `401.html` to prevent duplicate code. There's no need to create a new one and they can share the 401 title as `401-Unauthorized`.